Swope mce upgrade (#8473)
open-cluster-management/rhacm-docs: clusters/install_upgrade/upgrade_mce.adoc

> [#upgrading-mce]

Author: New content here

Author: checklist considered

= Upgrading {mce} 

Learn how to upgrade your {mce-short}. Control your {mce-short} upgrades by using the operator subscription settings in the {ocp-short} console.

*Required access:* {ocp-short} administrator

*Important:*

* Upgrades are always supported from the immediate previous version. You can upgrade to the next available feature release.
* If you are on a version with an *Extended Update Support* (EUS) status, you can upgrade to the next version, or you can skip to the next EUS version. 

> * You can perform skip-level upgrades from `<2.x>` to `<2.x+2>` on even number versions of {mce-short}, starting with version 2.11. For instance, when you upgrade {mce-short} 2.6, which is an EUS version, you can choose to upgrade to version 2.8, which is also an EUS version.

Reviewer: Should you specify ACM 2.11? 
```suggestion
* You can perform skip-level upgrades from `<2.x>` to `<2.x+2>` on even number versions of {mce-short}, starting with {acm-short} version 2.11. For instance, when you upgrade {mce-short} 2.6, which is an EUS version, you can choose to upgrade to version 2.8, which is also an EUS version.
```

Author: @dockerymick for a while we are calling our 2.11 because it is the first EUS version that the customer knows. The others we have are 2.13 and 2.15. (MCE releases are included) 

Because this is about and EUS upgrade, that is why we still talk about 2.11.

.Procedure

> You can upgrade your `MultiClusterEngine` from the console. The {olm} `operatorcondition` is used to control how versions are upgraded. When you initially deploy {mce-short} by using the operator, you make the following selections:

Reviewer: ```suggestion
You can upgrade your `MultiClusterEngine` resource from the console. Use the {olm} `operatorcondition` to control how versions are upgraded. When you initially deploy {mce-short} by using the operator, you make the following selections:
```

* *Channel:* Channel corresponds to the version of the product that you are installing. The initial channel setting is often the most current channel that was available at the time of installation.
* *Approval:* Approval specifies whether approval is required for updates within the channel, or if they are done automatically.
** If set to `Automatic`, then minor release updates in the selected channel are deployed without administrator intervention.

> ** If set to `Manual`, then each update to the minor release within the channel requires an administrator to approve the update.

Reviewer: To reduce the use of bullet points, did you want to use the approach that Alexandria showed us? 
```suggestion
Channel:: Channel corresponds to the version of the product that you are installing. The initial channel setting is often the most current channel that was available at the time of installation.
Approval:: Approval specifies whether approval is required for updates within the channel, or if they are done automatically.
- If set to `Automatic`, then minor release updates in the selected channel are deployed without administrator intervention.
- If set to `Manual`, then each update to the minor release within the channel requires an administrator to approve the update.
```
